routing/routing.go

```diff
@@ -140,12 +142,86 @@ type RouteFilter struct {
 	Index int
 }
 
+// Contains metrics used by LB algorithms
```

Comment:
Godoc please: LBMetrics contain ...
Same style required for all functions below

Reply:
Done.